common-app-lib/src/cmr/common_app/services/search/elastic_search_index.clj

```diff
@@ -267,7 +267,8 @@
         e-results (send-query-to-elastic context query)
         elapsed (- (System/currentTimeMillis) start)
         hits (get-in e-results [:hits :total :value])]
-    (info "Elastic query took" (:took e-results) "ms. Connection elapsed:" elapsed "ms")
+    (info "Elastic query for concept-type:" (:concept-type query) " and result format: " (:result-format query)
```

Comment:
FYI for others, there is a search dashboard that checks for this information in the log, the dashboard will need to be updated.

Reply:
can I change it to not need an update?